src/main/java/views/AcesUp/AcesUp.flt.html

```diff
@@ -139,29 +139,6 @@
 <script>
 
 var game;
-
-function display(game) {
-console.log(game);
-
-$( '.columnOfCards .cardLocation' ).html("");
-
-  $.each(game.cols[0], function( key, val ) {
-        $( '#c0 .l'+key ).html(val.value + " of " + val.suit);
```

Comment:
Suit doesn't exist anymore since the backend of the project was changed to have two suits (american and spanish).